examples/multiple_windows/lib/app/models.dart

```diff
@@ -61,18 +62,27 @@ class WindowManagerAccessor extends InheritedNotifier<WindowManager> {
   }
 }
 
+class TooltipSettings {}
+
 /// Settings that control the behavior of newly created windows.
 class WindowSettings {
   WindowSettings({
     this.regularSize = const Size(400, 300),
     this.dialogSize = const Size(200, 200),
+    this.positioner = const WindowPositioner(
+      parentAnchor: WindowPositionerAnchor.right,
+      childAnchor: WindowPositionerAnchor.left,
+    ),
```

Comment:
This seems pretty straightforward to use 👍 . I remember we talked a lot about WindowPositioner in the last PR.